### ISSUE TITLE
CS1574 is reported twice for the same node for records

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#53625]
**Version Used**: 

main

**Steps to Reproduce**:

https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzguATEBqAPgekIAIAeFAVzEgCcBPAPgFgAoQ/IgZQFMuiIAbFAHtSKHkQDG1LgDMAvACIAkjABuAqBgDC0mQvz0AdCzal8FKhDpNm0iUOoYiAJQAUsBESUBKIgG8iAF8gA

```csharp
/// <summary>
/// See also <see cref="InvalidCref"/>.
/// </summary>
record R(int I) { }
```

**Expected Behavior**:

One warning only.

**Actual Behavior**:

Two warnings.

```
warning CS1574: XML comment has cref attribute 'InvalidCref' that could not be resolved
warning CS1574: XML comment has cref attribute 'InvalidCref' that could not be resolved
```
